formats/polus-tabular-to-feather-plugin/run-plugin.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+#!/bin/bash
+version=$(<VERSION)
+#echo $(datapath)
+
+# Inputs
+inpDir=/data/inputs
+
+
+# Output paths
+outDir=/data/outputs
+
+# Log level, must be one of ERROR, CRITICAL, WARNING, INFO, DEBUG
+LOGLEVEL=INFO
+
+docker run -v /Users/mezukn/Desktop/polus/data:/data \
```

Comment:
You should replace the path on your local machine with an environment variable so that it's easy to change for someone else. I think in the original shell script I had an environment variable that would set a data path.